python/py-scipy/Portfile

```diff
@@ -60,8 +62,6 @@ if {${name} ne ${subport}} {
         # Work around thread local compiler selection bug. Remove after
         # https://github.com/macports/macports-base/pull/161 is released.
         compiler.blacklist-append {clang < 800}
-
-        compilers.allow_arguments_mismatch yes
```

Comment:
Won't removing this undo the fix for https://trac.macports.org/ticket/60520 ? I think this just needs to be moved for global SciPy use.

Reply:
upstream presumably [resolved](https://github.com/scipy/scipy/pull/11842) this issue in 1.5.0 (?); if that's indeed the case it should only be needed for the PY27/PY35 subports that are pinned to an earlier version.

Reply:
> upstream presumably [resolved](https://github.com/scipy/scipy/pull/11842) this issue in 1.5.0 (?)

This is according to https://docs.scipy.org/doc/scipy/reference/release.1.5.0.html:
>## Issues closed for 1.5.0
> * \#11611: Build failure with gfortran 10
>## Pull requests for 1.5.0
> * \#11842: BLD: fix build with gfortran 10

Reply:
I agree that removing the fix for the latest version is likely a matter of preference, at least right now. I don't know if upstream will eventually rework the affected code and will want scipy to be built without this flag; even then, it would presumably not hurt for it to be set anyway by MacPorts.

Reply:
Just tested & it looks like SciPy 1.5.0+ builds cleanly without this Fortran tweak ... so, looks like it's just needed for older versions. yay!